ui/src/containers/ParseUrl/index.tsx

```diff
@@ -1,30 +1,40 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom';
 import { useMst } from '../../store/root';
 import { Params } from '../../common/params';
+import { AuthCodeProps } from '../../store/auth';
 
 const ParseUrl: React.FC = () => {
-  const { resources } = useMst();
+  const { resources, user } = useMst();
+  const history = useHistory();
 
   if (window.location.search) {
     const searchParams = new URLSearchParams(window.location.search);
+    const status = searchParams.get('status');
+    const code = searchParams.get('code');
+
+    // It checks status and code and then redirect to authentication
+    if (status === '200' && code !== null) {
```

Comment:
What are we doing if status is not 200 ?

Reply:
Showing the error is alert message ?

Reply:
In case status is not 200 then user needs to retry login again and we are not showing any error or alert message

Reply:
I think we should some error in the form of alert in that case so that it becomes easier for the user to understand